src/command/output.rs

```diff
@@ -294,7 +294,14 @@ impl RoverOutput {
                 readme,
                 forum_call_to_action))
             }
-            RoverOutput::CheckResponse(check_response) => Some(check_response.get_table()),
+            RoverOutput::CheckResponse(check_response) => match check_response {
+                CheckResponse::OpeartionCheckResponse(operation_check_response) => {
+                    Some(operation_check_response.get_table())
+                }
+                CheckResponse::OpeartionLessCheckResponse(operation_less_check_response) => {
+                    Some(operation_less_check_response.to_output())
```

Comment:
if this truly should be an empty string, we should just return `None` here instead of `Some("")`

Reply:
Updated the string to be not just the empty string so going to leave this.